services/src/api/src/main/java/org/openkilda/northbound/dto/links/PathDto.java

```diff
@@ -17,31 +17,25 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import lombok.Value;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
-@JsonSerialize
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@Value
 public class PathDto {
 
     @JsonProperty("switch_id")
     private String switchId;
 
     @JsonProperty("port_no")
-    private int portNo;
+    private Integer portNo;
```

Comment:
What is the reason to use a wrapper class instead of a primitive? Do we support nulls for the fields?